applications/nrf_desktop/nrf_desktop_integration.rst

```diff
@@ -0,0 +1,953 @@
+.. _nrf_desktop_porting_guide:
+
+Integrating your own hardware
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section describes how to adapt the nRF Desktop application to different hardware, and lists steps required for adding a new board.
```

Comment:
Maybe we could link to configuration section from here. I think user must be aware of how nRF Desktop configuration works before starting porting (the sections are no longer next to each other). Something like:
```
.. tip:
   Make sure to get familiar with :ref:`nrf_desktop_configuration` section before porting the application to a new board.